src/tests/resource-manager-dispensers.test.ts

```diff
@@ -0,0 +1,64 @@
+import { RobotResourceManager, RobotResource } from '../resource-manager-robots';
+import fakeResources from '../mock/data/resources';
+import { DispenserResource, DispenserResourceManager } from '../resource-manager-dispensers';
+
+describe('Correct functionally of getIconPath', () => {
+  let manager: DispenserResourceManager;
+  let resourceData: Record<string, DispenserResource>;
+
+  beforeAll(() => {
+    resourceData = fakeResources().dispensers as Record<string, DispenserResource>;
+    manager = new DispenserResourceManager(resourceData);
+  });
+
+  test('dispenserName does not exists', () => {
```

Comment:
Can we have more descriptive test names? e.g. "returns falsy when dispenserName does not exist"